usr/share/rear/skel/OBDR/etc/scripts/system-setup.d/43-init-cciss.sh

```diff
@@ -1,7 +1,7 @@
 ### Disable OBDR mode
 ###
 
-if ! grep -q '^cciss ' /proc/modules; then
+if ! grep -q '^cciss ' -e '^hpsa' /proc/modules; then
```

Comment:
What is the reason that there is no trailing space character
in '^hpsa' in contrast to '^cciss ' that has one?

Reply:
You are correct, should be exact name of module. updated.